bins/src/bin/datadog-static-analyzer-server.rs

```diff
@@ -58,13 +58,25 @@ async fn serve_static(
     server_configuration: &State<ServerConfiguration>,
     name: &str,
 ) -> Option<NamedFile> {
-    let name_optional = server_configuration.files_to_serve.get(name);
+    server_configuration.static_directory.as_ref()?;
```

Comment:
This feels weird and hard to read. I understand you're doing it to check that `static_directory` is `Some`.

I would probably use a simple if and collide the 3 conditions (see suggestion below with the full code).